src/index.js

```diff
@@ -150,8 +155,101 @@ const renderLevel = (root, levelNumber) => {
         .join("")}
     </tr>
   `;
+
+  renderSettings = () => {  
+    const settingsHTML = `
+    <div id="modal">
+      <div class="modal-content">
+        <span class="close material-symbols-outlined">cancel</span>
+        <h1>Setting</h1>
```

Comment:
```suggestion
        <h1>Settings</h1>
```

As discussed, we might not need this menu?